qa-tests-backend/src/test/groovy/UpgradesTest.groovy

```diff
@@ -43,6 +43,11 @@ class UpgradesTest extends BaseSpecification {
             }
         }"""
 
+    private static final List<String> BROKEN_POLICIES = [
```

Comment:
the idea behind `KnownPolicyDiffs` is that it wouldn't fail if the specific sections were broken but it would fail if any other fields get broken.

For example, in this policy currently we know remediation and rationale are missing in default so we want to ignore that in the tests.

But let's say a migration was made to this policy that changed the _criteria_ but not for default policies (or vice versa). This would ignore that, but the other way using KnownPolicyDiffs wouldn't. 

Minor point, and I don't want to block since this is expected to be fixed by 73, but still worth pointing out.

Reply:
The fix for [ROX-13265](https://issues.redhat.com/browse/ROX-13265) should also remove this brute force `BROKEN_POLICIES` fix.